pkg/server/commands/connected_objects.go

```diff
@@ -155,7 +155,18 @@ func (c *ConnectedObjectsCommand) streamedConnectedObjects(
 	g := graph.BuildConnectedObjectGraph(c.Typesystem)
 
 	// find the possible incoming edges (ingresses) between the target user reference and the source (object, relation) reference
+	_, span2 := tracer.Start(ctx, "RelationshipIngresses", trace.WithAttributes(
```

Comment:
Another PR please? Let's follow up with this after the bug fix so that the changes herein are more apparently related to the bug fix.

Reply:
the spans that I added in this PR i used exclusively to debug this. But in the end they didn't prove very useful. I removed this. Let me know if I should remove the `tupleEval` span as well

Reply:
Yes, please do. The level of annotations on the traces feels very chatty.

Reply:
Agreed... if there's many tuples eval'ed this would be super noisy 🤦‍♀️